backend/src/test/java/com/soen341/instagram/dao/impl/CommentRepositoryTest.java

```diff
@@ -0,0 +1,58 @@
+package com.soen341.instagram.dao.impl;
+
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.model.Comment;
+import com.soen341.instagram.model.Picture;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class CommentRepositoryTest {
+    @Autowired
+    private CommentRepository commentRepository;
+    @Autowired
+    TestEntityManager testEntityManager;
+
+    private Comment comment;
+
+    @BeforeEach
+    public void beforeEach(){
+        commentRepository.deleteAll();
+    }
+
+    @Test
+    public void findByPicture(){
```

Comment:
Add the whole code for creating an account and picture in `beforeEach` so that you can add different tests (e.g. picture that has no comments)

Reply:
Same thing in `PictureRepositoryTest`